pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,13 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            raw_data = api_repr.get('data', b'')
+            padded_data = raw_data + b'=' * (4 - len(raw_data) % 4)
```

Comment:
That's actaully why the travis build was failing in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2527/commits/b8c4360c4268012de44733a0025aca13bdea801e.

So the current version just assumes that the console is broken and only tries to pad when there is an error when trying to decode it the first time.